es/index.js

```diff
@@ -175,7 +201,9 @@ export default function axiosRetry(axios, defaultOptions) {
         config.timeout = Math.max(config.timeout - lastRequestDuration, 1);
       }
 
-      return axios(config);
+      return new Promise((resolve) =>
```

Comment:
@moose56 I'd suggest to pass along the `error` as well, so one will be able to choose strategy according to the error, the same as happens with `retryCondition` so we get : 

```js
return new Promise(resolve => setTimeout(() => resolve(axios(config)), retryDelay(currentState.retryCount, error)));
```

Reply:
Good idea